doc/release/1.14.0-notes.rst

```diff
@@ -416,6 +416,10 @@ The printing style of ``np.void`` arrays is now independently customizable
 using the ``formatter`` argument to ``np.set_printoptions``, using the
 ``'void'`` key, instead of the catch-all ``numpystr`` key as before.
 
+unstructured void array's ``.item`` method now returns a bytes object
+---------------------------------------------------------------------
+``.item`` now returns a ``bytes`` object instead of a buffer or byte array.
```

Comment:
It's probably worth drawing attention to the fact that this new object is readonly, unlike the previous buffer or byte array that was read/write

Reply:
For that reason, this maybe should be under `Compatibility`

Reply:
On the other hand, the previous buffer or byte array wasn't a view or the original array or scalar. So writing to it had no effect. I'll add a note though.

Reply:
Indeed, but it means that users who expected to get a mutable scratch buffer no longer do so.

I suppose the alternative would be to return a `bytearray` object.

Reply:
I considered that too. But I suspect we would actually be causing bugs/confusion if we return a bytearray, since it vaguely implies it is a view when it isn't. Returning an immutable `bytes` makes clear you only got a copy.

Updated the note, by the way.